### PR TITLE
improve accessibility and clean up tailwind classes

### PR DESCRIPTION
**Pull request by JimmyHoenderdaal**
This is based on the colors an tweaks branche so we also merge inside there
This PR makes sure checout-theme is WCAG 2.1 AA approved with some accessibility fixes

### CODE REVIEW

File: resources/views/components/input/quantity.blade.php [hunk 1]
```diff
@@ -6,26 +6,28 @@
...
-            v-bind:step="item.qty_increments"
-            v-bind:dusk="'qty-'+index"
-        />
-        <button
-            class="flex-1 bg transition hover:bg-opacity-80"
-            v-on:click.prevent="variables.quantity = +variables.quantity + (item.product.stock_item?.qty_increments || 1);mutate()"
-        >+</button>
+    <form v-on:submit.prevent="mutate" class="w-20">
+        <label class="flex overflow-hidden border rounded">
+            <button
```
Comment (Roene-JustBetter) [inline resources/views/components/input/quantity.blade.php:11]: We don't need an aria-label here?

File: resources/views/components/input/quantity.blade.php
Comment (Roene-JustBetter) [inline resources/views/components/input/quantity.blade.php:1]: Maybe out of scope for this pull request. But can we use the quantity from Rapidez/core? https://github.com/rapidez/core/blob/master/resources/views/components/quantity.blade.php

**Review by Roene-JustBetter — CHANGES_REQUESTED** [review]

File: resources/views/components/input/quantity.blade.php
Comment (JimmyHoenderdaal) [reply resources/views/components/input/quantity.blade.php:1]: It's a bit out of scope and we already have a internal issues ready for this subject

File: resources/views/components/input/quantity.blade.php (hunk 1 above)
Comment (JimmyHoenderdaal) [reply resources/views/components/input/quantity.blade.php:11]: Didn't get any warnings from the accessibility devtool
But I suppose I can add it here

File: resources/views/checkout/pages/credentials.blade.php [hunk 2]
```diff
@@ -7,11 +7,11 @@
 @section('content')
     <div class="container">
         <x-rapidez-ct::layout class="lg:items-end">
-            <x-rapidez-ct::title-progress-bar :href="route('cart')" :$checkoutSteps :$currentStep :$currentStepKey>
+            <x-rapidez-ct::title-progress-bar :aria-label="__('Back to cart')" :href="route('cart')" :$checkoutSteps :$currentStep :$currentStepKey>
                 @lang('Credentials')
             </x-rapidez-ct::title-progress-bar>
             <x-slot:sidebar class="max-lg:hidden">
-                <a href="{{ url('/') }}" class="*:h-auto *:max-h-20 *:w-full *:object-contain">
+                <a href="{{ url('/') }}" aria-label="Home" class="*:h-auto *:max-h-20 *:w-full *:object-contain">
```
Comment (royduin) [inline resources/views/checkout/pages/credentials.blade.php:14]: Home should also be translatable I think?

**Review by royduin — CHANGES_REQUESTED** [review]

**Comment by royduin** [comment]
Conflicts 👀

**Review by Roene-JustBetter — APPROVED** [review]

**Review by jordythevulder — APPROVED** [review]

**Pull request merged**